### DISCUSSION TITLE
Please help, no sound?

### DISCUSSION THREAD

**Discussion by Tideryius** [elementary/triage#576] [Q&A]
Hello, I am an avid fan of Elementary OS, even back when it was just a theme pack. That said, I'm by no means an expert when it comes to Linux, especially when it pertains to hardware.

Essentially, everything works except for the sound. It says "no output device" detected. It seems nearly every other distro has this issue with this "janky" laptop except for the latest ubuntu. It has proper sound. I really want to go back to Elementary, but I need sound support to do so. I figure there may be some work around or fix to restore sound? I know it's based on ubuntu perhaps I could get what I need for it there? Sorry for being a noob with poor internet etiquette, as I said, I am new to this as well as this website. Please and thanks ahead of time.

**Comment by sudanchapagain** [comment]
this is completely random but i had this issue on NixOS. Installing SOF-firmware fixed it. <https://github.com/thesofproject/sof>

Issue could be related to alsa or pipewire. Maybe reinstall or something.
